lib/db.js

```diff
@@ -35,7 +35,30 @@ export default (connection_string) => {
               : resolve(result)
           ))
         )
-      );
+      )
+      .catch(err => {
+        const { message, position } = err;
+        if (message && position >= 0) {
+          const endLineWrapIndexOf = string.indexOf('\n', position);
+          const endLineWrapPos = endLineWrapIndexOf >= 0 ? endLineWrapIndexOf : string.length;
+          const stringStart = string.substring(0, endLineWrapPos);
+          const stringEnd = string.substr(endLineWrapPos);
+          const startLineWrapPos = stringStart.lastIndexOf('\n') + 1;
+          const padding = ' '.repeat(position - Math.max(startLineWrapPos, 0) - 1);
```

Comment:
`position` is guaranteed to be at least 0
`startLineWrapPos` is also guaranteed to be at least 0 (so the with Math.max doesn't seem to make sense?)
assuming `position` >= `startLineWrapPos`.
so overall the parameter passed to report is at least (0 - 0 - 1) = (-1).
' '.repeat(-1) throws exception both in browser and in node.js
so I think the Math.max test should be around all of that, i.e.
' '.repeat(Math.max(position - startLineWrapPos - 1, 0))

... or did I miss anything here?